### PR TITLE
Make sure to use entware xargs binary

### PR DESCRIPTION
**Pull request by mhummels**
Make sure to use the entware xargs binary located at `/opt/bin/xargs` instead of the busybox one `/usr/bin/xargs` which has only a reduced parameter set resulting in this error:
```
xargs: invalid option -- I
BusyBox v1.01 (2022.06.22-22:54+0000) multi-call binary

Usage: xargs [COMMAND] [OPTIONS] [ARGS...]

Executes COMMAND on every item given by standard input.

Options:
	-r	Do not run command for empty readed lines
	-x	Exit if the size is exceeded
	-0	Input filenames are terminated by a null character
	-t	Print the command line on stderr before executing it.
```

### CODE REVIEW

**Comment by Yannik** [comment]
Hi Michael,

thanks for opening this PR. Due to supporting various different qnap platforms, not all users have entware installed. 

However, I would be open to a PR which replaces `xargs` with a simple bash loop.

Yannik

**Comment by mhummels** [comment]
I dropped xargs and converted to a bash loop as suggested

File: renew_certificate.sh [hunk 1]
```diff
@@ -43,14 +43,17 @@ echo "Stopping Qthttpd hogging port 80.."
 
 /etc/init.d/Qthttpd.sh stop
 
-lsof -i tcp:80 -a -c python -t | xargs -r -I {} sh -c 'echo "Killing old python process {} hogging port 80" && kill {} && sleep 1'
+for PID in $(lsof -i tcp:80 -a -c python -t)
+do
+    echo "Killing old python process $PID hogging port 80" && kill $PID && sleep 1
```
Comment (Yannik) [inline renew_certificate.sh:48]: Can you split this up into one command per line?

**Review by Yannik — CHANGES_REQUESTED** [review]

File: renew_certificate.sh (hunk 1 above)
Comment (mhummels) [reply renew_certificate.sh]: Yes sure, done.

**Review by Yannik — APPROVED** [review]

**Pull request merged**